### PR TITLE
[TS] LPS-70777 Unable to login after removing the property passwords.encryption.algorithm.legacy

### PR DESCRIPTION
**Pull request by topolik**

### CODE REVIEW

**Comment by arthurchan35** [comment]
Hi @rodrigopaulino ,
It is a legit issue, but again it's part of what the password project is going resolve. The real problem is that when all the passwords are updated(from legacy to current, as the customer manually did test), these passwords will be all prepended with `{algorithmName}` , and if one removes the legacy property, no class will be handling the prepends.
However there is a little concern I have, the problem you will fixing is different than what LPS-70777 originally reports. What Alberto complains in LPS-70777 was portal does not handle password after removing legacy property (without updating passwords, see Mike Han's comment). BTW, we will have feature in password project to allow admin to force it.
To sum up, I personally would suggest customer to wait for the password project in future DXP release. Or if we really want to fix it right now, I think you should create a different ticket as what your customer reported is some different from LPS-70777.
I will also let other team member know about it.

Thanks

**Comment by rodrigopaulino** [comment]
I'll send another PR directly to you mentioning another ticket's number.
